README.md

```diff
@@ -5,6 +5,14 @@
 ### Dependencies
 - python3.6
 
+### Installation
+```bash
+pip3 install git+git://github.com/YouClid/youclid.git@master#egg=youclid
```

Comment:
If you try this now, it won't work, since this PR isn't merged into master yet. What you'd want is
```
pip3 install git+git://github.com/YouClid/youclid.git@feature/script-from-anywhere#egg=youclid
```